feature-libs/customer-ticketing/styles/components/_customer-ticketing-list.scss

```diff
@@ -84,28 +84,39 @@
     }
   }
 
+  .cx-ticketing-list-data-extra {
```

Comment:
There's no need to have this 'extra' class.

You can re-use the cx-ticketing-list-data as done before, but approach it using the way I mentioned here in terms of what you had with all the spans everywhere before https://github.com/SAP/spartacus/pull/16059#discussion_r993686347